Packs/AzureStorageContainer/ReleaseNotes/1_1_0.md

```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### Azure Storage Container
+This is a update to existing Azure storage Container integration with a new command.
```

Comment:
```suggestion
This is an update to existing Azure storage Container integration with a new command.
```
Also please elaborate on which command was added

Reply:
can you please suggest where this message is written on integration, the new command added is 
azure-storage-container-sas-create

Reply:
> 

@Geekverma This is the release notes in there we will have the information on what we have changed in the current version, so please try to be elaborate as possible, people who use Demisto will see those release notes when they'll update the integration and we want it to be as clear as possible as to what have changed